src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -113,6 +121,14 @@ public function searchGridAction(Request $request)
      */
     public function createAction(Request $request)
     {
+        if ($this->get('prestashop.adapter.multistore_feature')->isActive()) {
+            $this->addFlash(
+                'info',
+                $this->trans('Note that this feature is available in all shops context only. It will be added to all your stores.',
+                    'Admin.Notifications.Info')
```

Comment:
```suggestion
                $this->trans(
                    'Note that this feature is available in all shops context only. It will be added to all your stores.',
                    'Admin.Notifications.Info'
                )
```